examples/poisson_2d_multi_patch.py

```diff
@@ -0,0 +1,207 @@
+
+from sympy.core.containers import Tuple
+from sympy                 import Matrix               
+from sympy                 import Function                                
+from sympy                 import pi, cos, sin, exp                        
+      
+from sympde.core     import Constant
+from sympde.calculus import grad, dot, inner, rot, div
+from sympde.calculus import laplace, bracket, convect
+from sympde.calculus import jump, avg, Dn, minus, plus
+
+from sympde.topology import ScalarFunctionSpace
+from sympde.topology import element_of, elements_of
+from sympde.topology import InteriorDomain, Union
+from sympde.topology import Boundary, NormalVector
+from sympde.topology import Domain
+from sympde.topology import trace_1
+from sympde.topology import Square
+from sympde.topology import ElementDomain
+from sympde.topology import Area
+from sympde.topology import IdentityMapping, PolarMapping, AffineMapping
+                         
+from sympde.expr.expr          import LinearExpr
+from sympde.expr.expr          import LinearForm, BilinearForm
+from sympde.expr.expr          import integral              
+from sympde.expr.expr          import Functional, Norm                       
+from sympde.expr.expr          import linearize                      
+from sympde.expr.evaluation    import TerminalExpr
+from sympde.expr               import find, EssentialBC
+
+from psydac.api.discretization import discretize
+from psydac.fem.vector         import VectorFemField
+from psydac.fem.basic          import FemField
+from psydac.utilities.utils    import refine_array_1d
+
+from mpi4py import MPI
+#==============================================================================
+
+def run_poisson_2d(solution, f, domain, mappings, ncells, degree, comm=None):
+
+    #+++++++++++++++++++++++++++++++
+    # 1. Abstract model
+    #+++++++++++++++++++++++++++++++
+
+    V   = ScalarFunctionSpace('V', domain, kind=None)
+
+    u, v = elements_of(V, names='u, v')
+    nn   = NormalVector('nn')
+
+    bc   = EssentialBC(u, solution, domain.boundary)
+
+    error  = u - solution
+
+    I = domain.interfaces
+
+    kappa  = 10**9
+
+   # expr_I =(
+   #         - dot(grad(plus(u)),nn)*minus(v)  + dot(grad(minus(v)),nn)*plus(u) - kappa*plus(u)*minus(v)
+   #         + dot(grad(minus(u)),nn)*plus(v)  - dot(grad(plus(v)),nn)*minus(u) - kappa*plus(v)*minus(u)
+   #         - dot(grad(plus(v)),nn)*plus(u)   + kappa*plus(u)*plus(v)
+   #         - dot(grad(minus(v)),nn)*minus(u) + kappa*minus(u)*minus(v))
+
+    expr_I =- 0.5*dot(grad(plus(u)),nn)*minus(v)  + 0.5*dot(grad(minus(v)),nn)*plus(u)  - kappa*plus(u)*minus(v)\
+            + 0.5*dot(grad(minus(u)),nn)*plus(v)  - 0.5*dot(grad(plus(v)),nn)*minus(u)  - kappa*plus(v)*minus(u)\
+            - 0.5*dot(grad(minus(v)),nn)*minus(u) - 0.5*dot(grad(minus(u)),nn)*minus(v) + kappa*minus(u)*minus(v)\
+            - 0.5*dot(grad(plus(v)),nn)*plus(u)   - 0.5*dot(grad(plus(u)),nn)*plus(v)   + kappa*plus(u)*plus(v)
```

Comment:
Do we have a way to write these expressions using jumps and averages?

Reply:
I have not tried it yet but it might work

Reply:
no it doesn't work we will need to fix it in `Sympde`

Reply:
Could you create an issue in Sympde about it?

Reply:
okay